config/routes.rb

```diff
@@ -5,10 +5,10 @@
   get '/pages/guidance', to: redirect('/about-bt-wifi')
 
   resources :application_forms do
-    get 'success/:application_form_id', to: 'application_forms#success', as: :success
+    get 'success/:recipient_id', on: :collection, to: 'application_forms#success', as: :success
```

Comment:
Minor nit, but in terms of modelling the resources it looks like the routes could be done as so:

```ruby
  resources :recipients, only: [] do
     resource :application_form, only: [] do
       get :success
     end
  end
```

Giving the route:

```
success_recipient_application_form GET    /recipients/:recipient_id/application_form/success(.:format)             application_forms#success
```

That seems more accurate, but I don't know if the path chosen, `/applications_forms/:recipient_id/success`, is intentional ... but it does seem strange at first glance to key application forms by the recipient id.

Reply:
Yeah, it's a hangover artefact from the first draft, which mixed in user details with recipient details. Since we moved to a simpler model of requiring sign-in, we arguably don't even need the _form abstraction any more.... but I want to wait until I've had chance to think about implementing the latest designs on the design prototype first before refactoring it out